x-pack/plugins/alerting/server/task_runner/task_runner.test.ts

```diff
@@ -1441,29 +1440,31 @@ describe('Task Runner', () => {
         true
       );
       actionsClient.ephemeralEnqueuedExecution.mockResolvedValue(mockRunNowResponse);
-      ruleType.executor.mockImplementation(
-        async ({
-          services: executorServices,
-        }: RuleExecutorOptions<
-          RuleTypeParams,
-          RuleTypeState,
-          AlertInstanceState,
-          AlertInstanceContext,
-          string,
-          RuleAlertData
-        >) => {
-          executorServices.alertFactory.create('1').scheduleActions('default');
-          return { state: {} };
-        }
-      );
+      ruleType.executor.mockImplementation(async () => {
+        return { state: {} };
+      });
 
-      getSummarizedAlertsMock.mockResolvedValue({
-        new: {
-          count: 1,
-          data: [mockAAD],
-        },
-        ongoing: { count: 1, data: [] },
-        recovered: { count: 0, data: [] },
+      alertsService.createAlertsClient.mockImplementation(() => {
+        return {
```

Comment:
can we use the  mock alerts client from `alerts_client.mock.ts` here?

Reply:
done